_posts/2017-02-21-ome-files-cpp-0-3-1.md

```diff
@@ -76,7 +76,7 @@ Note that this is a work in progress and subject to change.
 
 **Errata**
 
-See [Errata](https://www.openmicroscopy.org/site/support/ome-files-cpp/ome-cmake-superbuild/manual/html/errata.html) for this release.
+See [Errata](https://docs.openmicroscopy.org/ome-cmake-superbuild/0.3.1/errata.htm) for this release.
```

Comment:
Also wondering, is there no automated link checking to pick this type of error up?